src/Compilers/Core/Portable/Operations/IBinaryOperatorExpression.cs

```diff
@@ -32,5 +32,13 @@ public interface IBinaryOperatorExpression : IHasOperatorMethodExpression
         /// value types.
         /// </summary>
         bool IsLifted { get; }
+        /// <summary>
+        /// <code>true</code> if this is a 'checked' binary operator.
+        /// </summary>
+        bool IsChecked { get; }
+        /// <summary>
+        /// <code>true</code> if the comparison is text based for string or object comparison in VB.
+        /// </summary>
+        bool IsCompareText { get; }
```

Comment:
Yes, @AlekseyTs explicitly requested this in https://github.com/dotnet/roslyn/issues/8846